api/oauth.go

```diff
@@ -152,24 +152,27 @@ func allowOAuth(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	c.LogAudit("attempt")
 
-	w.Header().Set("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
Actually, if you're just setting it to `application/json` I'm pretty sure you can just remove this line since we set it to that by default I think